package.json

```diff
@@ -8,7 +8,7 @@
     "precommit": "jscs . --fix && eslint ./client/app && npm test",
     "postinstall": "npm run build",
     "dev-client": "gulp",
-    "dev-server": "env $(cat .env) nodemon server/index.js --exec babel-node",
+    "dev-server": "nf run nodemon server/index.js",
```

Comment:
The way Heroku starts the server with `nf` is to check for a `Procfile`.

In the other VR app, we have a Procfile of:
```
web: node server.js
dev: nodemon server.js
```

Heroku will pick up that it's a node app and will run `$ nf start web=1`. To keep our envs similar, can you create the `Procfile` similar to the above but with `dev: nodemon server/index.js`?

Then the `package.json` can be: `"dev-server": "nf start dev=1"`. This will align it well!